pkgm.ts

```diff
@@ -212,8 +212,7 @@ async function shim(args: string[], basePath: string) {
       for await (const entry of Deno.readDir(bin_prefix)) {
         if (!entry.isFile && !entry.isSymlink) continue;
         const name = entry.name;
-        const shim = "#!/bin/sh\n\n" +
-          `exec ${pkgx} +${pkg.project}=${pkg.version} -- ${name} "$@"\n`;
```

Comment:
I think you should keep the `+pkg` part. Also, you could also keep `--`.

Reply:
probably a good idea, actually.